packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -110,14 +116,16 @@ export class FilamentViewer extends LitElement {
 
   private[$initialize]() {
     this[$canvas] = this.shadowRoot!.querySelector('canvas');
-    const engine = Engine.create(this[$canvas]!);
+    const engine = Engine.create(this[$canvas]!, {alpha: true});
     const view = engine.createView();
 
+    const entityManager = EntityManager.get();
+    const emptyEntity = entityManager.create();
     this[$engine] = engine;
     this[$scene] = engine.createScene();
     this[$swapChain] = engine.createSwapChain();
     this[$renderer] = engine.createRenderer();
-    this[$camera] = engine.createCamera();
+    this[$camera] = engine.createCamera(emptyEntity);
```

Comment:
Not sure what entity im supposed to put into camera's constructor. Is it ok to just put an empty entity?

Reply:
Yes this is correct.  We're trying to make our ECS more self-consistent; here you're adding a camera component to a blank entity.